input/templates/partials/catalog/product.json

```diff
@@ -1,119 +1,137 @@
+                    "value": "red",
+                    "selected": false
                 },
                 {
-                    "label" : "Navy Blue",
-                    "value" : "blue",
-                    "selected" : true
+                    "label": "Navy Blue",
+                    "value": "blue",
+                    "selected": true
                 },
                 {
-                    "label" : "Green",
-                    "value" : "green",
-                    "selected" : false
+                    "label": "Green",
+                    "value": "green",
+                    "selected": false
                 }
             ],
             "selectData": {
                 "red": {
                     "size": [
-                        "S", "M", "L"
+                        "S",
+                        "M",
+                        "L"
                     ]
                 },
                 "blue": {
                     "size": [
-                        "S", "L"
+                        "S",
+                        "L"
                     ]
                 },
                 "green": {
                     "size": [
-                        "S", "M"
+                        "S",
+                        "M"
                     ]
                 }
             }
         },
         {
             "name": "Size",
             "key": "size",
-            "list" : [
+            "list": [
                 {
-                    "label" : "S",
-                    "value" : "S",
-                    "selected" : false
+                    "label": "S",
+                    "value": "S",
+                    "selected": false
                 },
                 {
-                    "label" : "M",
-                    "value" : "M",
-                    "selected" : true
+                    "label": "M",
+                    "value": "M",
+                    "selected": true
                 },
                 {
-                    "label" : "L",
-                    "value" : "L",
-                    "selected" : false
+                    "label": "L",
+                    "value": "L",
+                    "selected": false
                 }
             ],
             "selectData": {
                 "S": {
                     "color": [
-                        "red", "blue", "green"
+                        "red",
+                        "blue",
+                        "green"
                     ]
                 },
                 "M": {
                     "color": [
-                        "red", "green"
+                        "red",
+                        "green"
                     ]
                 },
                 "L": {
                     "color": [
-                        "red", "blue"
+                        "red",
+                        "blue"
                     ]
                 }
             }
         }
-    ]
+    ],
+    "availability": "available"
```

Comment:
In any case, we have here mixed concerns between product and variant and we still have to re-structure the whole thing, so let's have that in mind at that time.